### DISCUSSION TITLE
Have a unit test that connectors have to pass an minimum bar

### DISCUSSION THREAD

**Discussion by alberttwong** [slingdata-io/sling-cli#182] [General]
Thoughts?

**Comment by flarco** [comment]
Yes indeed, I have my tests that mainly hit the all required ELT operations.
So basically e2e for running sling command for reading/writing to connectors in full-refresh and incremental modes.
See [here](https://github.com/slingdata-io/sling-cli/blob/main/cmd/sling/tests/tasks.tsv).
Unit testing is better, but requires a high amount of time investment to implement. I wish AI could do that for me already!
